simulator/scheme-adapter.env

```diff
@@ -54,9 +54,9 @@ CACHE_EXPIRY_SECONDS=3600
 
 # SWITCH ENDPOINT
 # The option 'PEER_ENDPOINT' has no effect if the remaining three options 'ALS_ENDPOINT', 'QUOTES_ENDPOINT', 'TRANSFERS_ENDPOINT' are specified.
-PEER_ENDPOINT=mojaloop-testing-toolkit:5000
-#PEER_ENDPOINT=172.17.0.1:5000
-#PEER_ENDPOINT=docker.for.mac.localhost:5000
+PEER_ENDPOINT=mojaloop-testing-toolkit:4040
+#PEER_ENDPOINT=172.17.0.1:4040
```

Comment:
Please add reasons for commented-out lines here, or otherwise remove them. Applies to other lines below...


```suggestion
## <REASON FOR THIS TO BE HERE>
#PEER_ENDPOINT=172.17.0.1:4040
```